src/main/java/at/aau/se2/handler/game/subhandlers/SwitchCardPlayerHandler.java

```diff
@@ -1,16 +1,91 @@
 package at.aau.se2.handler.game.subhandlers;
 
+import at.aau.se2.exceptions.PlayerNotFoundException;
+import at.aau.se2.handler.game.GameHandler;
+import at.aau.se2.model.Actioncard;
 import at.aau.se2.utils.Lobby;
+import at.aau.se2.utils.Player;
+import at.aau.se2.utils.UtilityMethods;
 import com.fasterxml.jackson.databind.JsonNode;
+import org.springframework.web.socket.TextMessage;
 import org.springframework.web.socket.WebSocketSession;
 
+import java.io.IOException;
+import java.util.List;
+import java.util.logging.Logger;
+
 public class SwitchCardPlayerHandler implements ActionHandler {
+
+    int passiveanswer;
+
     @Override
     public void handleMessage(WebSocketSession session, JsonNode msg, Lobby lobby){
-        // communication between two players
+        try {
+            String[] textfrommessage= messageContent(msg);
+            String usernametoswitchwith=textfrommessage[1];
```

Comment:
Variablenname zu lang & CamelCase verwenden!
```suggestion
            String switchUsername = msgContent[1];
```